src/modules/shopify-app-install/shopify-app-install.repository.ts

```diff
@@ -0,0 +1,67 @@
+import '@shopify/shopify-api/adapters/node';
+import { Injectable } from "@nestjs/common";
+import {
+  shopifyApi,
+  ApiVersion,
+  Shopify,
+  BeginParams,
+} from '@shopify/shopify-api';
+import { Request, Response } from 'express';
+import { restResources } from '@shopify/shopify-api/rest/admin/2023-07';
+import { ConfigService } from '@nestjs/config';
+import { ShopifyConfig } from '@config/shopify.config';
+
```

Comment:
too many any in this file

Reply:
Changed some but for session there will be type when we'll be ready to add webshops.